project1/application.py

```diff
@@ -18,24 +21,71 @@
 app.config["SESSION_TYPE"] = "filesystem"
 Session(app)
 
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.app_context().push()
+
+db.init_app(app)
+
+db.create_all()
+
+
 # Set up database
-engine = create_engine(os.getenv("DATABASE_URL"))
-db = scoped_session(sessionmaker(bind=engine))
+# engine = create_engine(os.getenv("DATABASE_URL"))
+# db = scoped_session(sessionmaker(bind=engine))
 
 @app.route("/")
 def index():
-    return render_template("main.html")
+    return render_template("index.html")
+
+@app.route("/admin")
+def admin():
+    users = User.query.order_by("timestamp").all()
+    return render_template("admin.html", users = users)
+    
 
 @app.route("/register", methods = ['GET','POST'])
 def register():
-    
     if (request.method=="POST"):
+        # data.query.all()
+        name = request.form.get("name")
+        print(name)
+        password = request.form.get("Password")
+        print(password) 
+        contact = request.form.get("ContactNumber")
+        print(contact)
+        regist = User(username=name, password=password)
+        if User.query.get(name):
+            return render_template("register.html",name1=name)
+        db.session.add(regist)
+        db.session.commit()
+        return render_template("register.html",name=name)
+    return render_template("register.html")
```

Comment:
send a message indicating successfull or not

Reply:
okay madam